rotkehlchen/tests/unit/test_evm_contracts.py

```diff
@@ -33,3 +64,71 @@ def test_evm_abi_data(globaldb):
             assert serialized_abi == entry[1]
             assert entry[1] not in abis_set
             abis_set.add(entry[1])
+
+
+def test_get_remote_abi(ethereum_inquirer: 'EthereumInquirer'):
+    """Tests that pulling an ABI from rotki data repository work"""
+    def requests_mock(url, timeout):
+        if url == f'https://raw.githubusercontent.com/rotki/data/develop/contracts/abi/{TEST_ABI_NAME}.json':  # noqa: E501
+            return MockResponse(status_code=200, text=json.dumps(TEST_ABI))
+
+        raise AssertionError(f'Unexpected url {url} was called')
+
+    with patch('requests.get', side_effect=requests_mock):
+        assert ethereum_inquirer.contracts.abi(TEST_ABI_NAME) == TEST_ABI  # type: ignore[call-overload]  # noqa: E501
```

Comment:
why this type ignore?

Reply:
`.abi` function for agument `name` supports only values from literal `ETHEREUM_KNOWN_ABI` and `TEST_ABI_NAME` is not a part of that literal.